msal/oauth2cli/oauth2.py

```diff
@@ -214,9 +213,9 @@ def initiate_device_flow(self, scope=None, timeout=None, **kwargs):
         if not self.configuration.get(DAE):
             raise ValueError("You need to provide device authorization endpoint")
         flow = self.session.post(self.configuration[DAE],
-            data={"client_id": self.client_id, "scope": self._stringify(scope or [])},
-            timeout=timeout or self.timeout,
-            **kwargs).json()
+                                 data={"client_id": self.client_id, "scope": self._stringify(scope or [])},
```

Comment:
There seems no substantial changes at all. I think you can simply revert changes to this entire file.

Reply:
No changes to logic, but fixes spacing and indentation that breaks PEP 8. I reverted changes, you can update that later.